src/utils/image_build.py

```diff
@@ -21,11 +21,7 @@ def build_and_upload_images(
     arches: str,
 ):
     if not build_args:
-        build_args = (
-            '"--disable-tests --with-incompatible-bdb --without-gui --disable-bench '
-            "--disable-fuzz-binary --enable-suppress-external-warnings "
-            '--without-miniupnpc --without-natpmp"'
```

Comment:
Don't need these as we don't install the deps for them in the image, so they don't get build automatically.